HTML-CSS/figma-project2/css/index.css

```diff
@@ -0,0 +1,220 @@
+
+/* *************************COMMON CSS******************* */
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.title {
+    font-size: 2rem;
+    color: #252525;
+}
+
+.text-left {
+    text-align: left;
+}
+
+.text-justify {
+    text-align: justify;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.text {
+    font-size: 1rem;
+    color: grey ;
+    font-weight: 500;
+    
+}
+
+
+
+.d-flex{
+    display: flex;
+}
+
+.justify-between{
+    justify-content: space-between;
+}
+
+.btn {
+    background-color: #405cf5;
+    border-radius: 6px;
+    border-width: 0;
+    color: #fff;
+    cursor: pointer;
+    font-size: 100%;
+    text-align: center;
+  }
+
+
+body {
+    background-image: linear-gradient( rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3) ), url('../images/bg-image.jpg');
+    background-attachment: fixed;
+}
+
+.img-fluid {
+    object-fit: cover;
+    max-width:100%;
+    max-height:100%;
+    
+}
+
+/* *************************HEADER CSS******************* */
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    background-color: white;
+    position: sticky;
+    top: 0px;
+    
+    
+}
+
+.header-item {
+    margin: 1rem 13%;
+    padding: 5px 10px;
+}
+
+/* *************************HOME CSS******************* */
+  
+.main-container {
```

Comment:
Bazı ekranlar çok büyük olabilliyor :D Bu yüzden main container için oransal bir width değeri, max-width verebilirsin static px olarak. Sonrasında containerini ortaladığında bütün ekranlarda herhangi bir bozunma olmadan tasarımını koruyabilirsin.